frontend/packages/component-library/src/carousel/carousel.module.scss

```diff
@@ -18,6 +18,11 @@
   // Added min-height so there's workable space in the Contentful experiences editor,
   // it won't have an effect on anything visually once slides are added.
   min-height: 50px;
+
+  :global(.swiper-slide) {
+    display: flex;
+    height: auto;
+  }
```

Comment:
This was added so the action blocks stretch to match the tallest one in the group.